doc/api/buffer.markdown

```diff
@@ -541,6 +541,42 @@ console.log(bufA.length);
 // 42
 ```
 
+### Class Method: Buffer.encode(str[, str_encoding][, target_encoding])
```

Comment:
Ah, sorry, I mean: Currently, it is not clear that `Buffer.encode('test', 'base64')` specifies the `target_encoding`, not the `str_encoding`, is it?